src/app/routes/utils/regex/utils/index.js

```diff
@@ -48,7 +48,7 @@ export const getCpsAssetRegex = services => {
 
 export const getLegacyAssetRegex = services => {
   const serviceRegex = getServiceRegex(services);
-  return `/:service(${serviceRegex})/:assetUri(${legacyAssetUriRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
+  return `/:service(${serviceRegex}):variant(${variantRegex})?/:assetUri(${legacyAssetUriRegex})?:amp(${ampRegex})?`;
 };
```

Comment:
For Legacy Asset paths, variant should be after the service, e.g. 
`https://www.test.bbc.com/ukchina/simp/multimedia/2015/11/151120_video_100w_london_chinese_entrepreneurs`

Reply:
I think this should also be the case for CPS assets: https://www.bbc.com/serbian/cyr/svet-51656881

Reply:
@tochwill This PR [has been updated](https://github.com/bbc/simorgh/pull/5718/files#diff-0e75337cdc4f57576789fbe816ebd2d7R46) to have this fixed.